_maps/map_files/BigRed_v2/BigRed_v2.dmm

```diff
@@ -15035,15 +15030,13 @@
 /area/bigredv2/caves/lambda_lab)
 "aSu" = (
 /obj/structure/barricade/wooden{
-	desc = "This barricade is heavily reinforced. Nothing short of blasting it open seems like it'll do the trick, that or melting the breams supporting it...";
-	health = 25000
+	desc = "This barricade is heavily reinforced. Nothing short of blasting it open seems like it'll do the trick, that or melting the breams supporting it..."
```

Comment:
Will require either a `/obj/structure/barricade/wooden/reinforced` subtype or the description removed as well.
Sort of a mapper issue.